src/setup/ConnectToRemote.tsx

```diff
@@ -83,10 +89,17 @@ const ConnectToRemote = inject(SETTINGS_STORE)(
                     id="ip-port"
                     label="IP:Port"
                     variant="outlined"
-                    value={ipAndPort}
+                    value={ipAndPortWithoutHttp}
+                    InputProps={{
+                      startAdornment: (
+                        <InputAdornment position="start">
+                          https://
+                        </InputAdornment>
+                      ),
+                    }}
                     onChange={(e) => {
                       setConnectionFailed(false);
-                      setIpAndPort(e.target.value);
+                      setIpAndPort(`https://${e.target.value}`);
```

Comment:
Just a minor suggestion: maybe we could avoid adding the `https://` twice if we would either 
a. leave the `ipAndPort` value without `https://`;
b.  pass the `ipAndPort` value as `xudDockerUrl` parameter into `handleConnectClick` and remove the `address` variable in that function as it could just be replaced with `xudDockerUrl` without needing to add the prefix.